CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # spring-service
 
+## 3.3.1
```

Comment:
Nit: Personally I would consider this more of a larger backwards-compatible code improvement and increment minor version instead

Reply:
right, fixed!